<launch>
  <arg name="message_display" default="screen" doc="output type"/>
  
  <node pkg="mt4n" type="mt4n" name="mt4n" output="$(arg message_display)">
    <param name="serial_port" value="/dev/modbus" />
    <param name="baud_rate" value="9600" />
    <param name="slave_num" value="11" />
    <param name="node_name" value="mt4n" />
  </node>
</launch>
